### DISCUSSION TITLE
How to toggle a widget?

### DISCUSSION THREAD

**Discussion by TornaxO7** [elkowar/eww#574] [Q&A]
Hello! I've created a dashboard for me but I'd like to toggle it to me needs. `eww help` doesn't provide a `eww toggle <window>` command if I'm seeing it right. What are the alternatives?

**Comment by TornaxO7** [comment]
Just wanted to try if I can add a toggle command until I saw [the toggle arg](https://github.com/elkowar/eww/blob/master/crates/eww/src/opts.rs#L117) for `eww open` ._.

**Reply by TornaxO7** [reply]
Wait, if I use the `toggle` arg and reopen my window, then the [graph](https://elkowar.github.io/eww/widgets.html#graph) widget will be resetted. Does anyone has a suggestion how to avoid this?

**Reply by viandoxdev** [reply]
Well you could hide the whole window (see #517 ) (and have a transparent background in the scss obviously) instead of closing it, but it would pickup mouse events if it isn't in the background

**Reply by TornaxO7** [reply]
Hm.... sadly that's not really my intention :(
